pwnlib/util/misc.py

```diff
@@ -4,6 +4,7 @@
 import errno
 import os
 import platform
+import pprint
```

Comment:
This import should be included in a different file, under `pwntools/docs/`, if not used by code, like the following imports:
https://github.com/Gallopsled/pwntools/blob/586e04a8028f7a51b79ec516ac755e2b5ccbb4a6/docs/source/util/misc.rst#L1-L4

It also probably should be:
```py
from pprint import pprint
```
to ensure that modified doctests are easy to read.

Reply:
Changed as requested.